speechbrain/augment/augmenter.py

```diff
@@ -176,6 +184,9 @@ def forward(self, x, lengths):
             input to augment.
         lengths : torch.Tensor
             The length of each sequence in the batch.
+        total_augmentation : int
+            The total number of augmentations perform for each signal
+            (with the orignal signal included in the count).
```

Comment:
It seems like the motivation for this return value is to record how many copies of the data there are when `parallel_augment` is enabled. But this wouldn't change from run to run, would it? I'm wondering if this is better done as a property or method (e.g. `augmenter.get_num_augmentations()` or `augmenter.get_output_replication_count()` or something like this). If you want to keep it here this should be in a `Returns` section rather than `Arguments`.

Reply:
Returning this variable is needed to figure out in train.py how many augmentations we apply to the input batch and augment the labels accordingly. For instance, if we apply 3 augmentations in parallel, the label tensor should be augmented by a factor of 3. It can definitely be computed with a specific method, or simply access it with augmenter.num_augmentations.

Reply:
I wonder if we could simply pass the labels here too. Or have another method like `replicate_labels` that takes care of it.

Reply:
Yes, we can do that actually!